packages/boilerplate-generator/template-web.browser.js

```diff
@@ -9,8 +9,9 @@ export const headTemplate = ({
 }) => {
   var headSections = head.split(/<meteor-bundled-css[^<>]*>/, 2);
   var cssBundle = [...(css || []).map(file =>
-    template('  <link rel="stylesheet" type="text/css" class="__meteor-css__" href="<%- href %>">')({
+    template('  <link rel="stylesheet" type="text/css" class="__meteor-css__" href="<%- href %>" integrity="sha512-<%- hash %>" crossorigin="anonymous">')({
```

Comment:
I'm curious if this `crossorigin="anonymous"` might potentially cause problems for existing deployments which might necessitate the presence of user credentials (as defined [here](https://www.w3.org/TR/cors/#user-credentials); e.g. HTTP Auth, Cookies, etc.) in order to serve the JS or CSS files (let's say the Meteor's deployment was fully behind that forced HTTP Authentication).

From what I understand, it's mandatory for `crossorigin` to be set for SRI to work (See [0](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity), [1](https://www.npmjs.com/package/ember-cli-sri#crossorigin-attribute)), but I'm afraid this might need to be toggleable between [the valid options](https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_settings_attributes), `anonymous` and `use-credentials`.

Any thoughts?

Reply:
Having this configurable sounds good to me. But where would this config go ideally? Any ideas?